tests/expected_report.txt

```diff
@@ -35,19 +35,20 @@ tests/input/semicolon_spacing.php                     3       0
 tests/input/single-line-array-spacing.php             5       0
 tests/input/spread-operator.php                       6       0
 tests/input/static-closures.php                       1       0
+tests/input/strings.php                               1       0
```

Comment:
I doubt this has to do with your changes, right? If not, it should be in a separate commit.

Reply:
This PR fixes running tests on unpatched versions and `strings.php` is run there too. This was missing in the tests now, see https://github.com/doctrine/coding-standard/runs/7079914164?check_suite_focus=true

Reply:
Oh ok sorry I misunderstood the purpose of this PR